release.md

```diff
@@ -1,29 +1,29 @@
 # Release Procedure
 
-
 ## Versioning
 
-Pulser version follows the [Semantic Versioning 2.0.0 specifcation](https://semver.org/spec/v2.0.0.html), which means its versions are numbered as MAJOR.MINOR.PATCH. 
-
-Currently (as of July 2022), Pulser is still in an early development phase and has a MAJOR of 0 - as such, both breaking and backwards compatible changes and additions to the API will be introduced through increments in the MINOR until version 1.0.0 is released. 
+Pulser version follows the [Semantic Versioning 2.0.0 specifcation](https://semver.org/spec/v2.0.0.html), which means its versions are numbered as MAJOR.MINOR.PATCH.
 
-During this phase, only two type of releases are envisioned:
+According to this specification, three types of releases are envisioned:
 
-- A scheduled release, where the MINOR is bumped and the PATCH is reset (`0.{x}.{y} -> 0.{x+1}.0`)
-- A hotfix, where the PATCH is bumped (`0.{x}.{y} -> 0.{x}.{y+1}`)
+- A major release, where breaking changes may be introduced. Here, the MAJOR is bumped and the MINOR and PATCH are reset (`{x}.{y}.{z} -> {x+1}.0.0`)
+- A minor release, where only the backwards compatible changes are added. Here, the MINOR is bumped and the PATCH is reset (`{x}.{y}.{z} -> {x}.{y}0`)
```

Comment:
Isn't there couple of thing missing in the last line ?
```suggestion
- A minor release, where only the backwards compatible changes are added. Here, the MINOR is bumped and the PATCH is reset (`{x}.{y}.{z} -> {x}.{y+1}.0`).
```

Reply:
You're right, thanks!